app/src/main/java/com/example/bookwormadventuresdeluxe2/UserObject.java

```diff
@@ -0,0 +1,83 @@
+package com.example.bookwormadventuresdeluxe2;
+
+/**
+ * UserObject is required to display information pulled from Firebase
+ */
+public class UserObject
+{
+    private String username;
+    private String email;
+    private String phoneNumber;
+    private String userId;
+    private String documentId;
+
+    public UserObject()
+    {
+
```

Comment:
Just for personal understanding, this empty constructor is for Firebase. Right?

Reply:
yes